src/mtx/utils/CommandBinaryBuffer.ts

```diff
@@ -707,7 +707,11 @@ export interface IEnergyPeriod {
 export interface IEnergies extends Array<types.TUint32 | null> {}
 
 export interface IPackedEnergiesWithType {
-    energyType?: TEnergyType,
+    /**
+     * `1` - `A+`, `2` - `A-`
```

Comment:
would be nice to have @link to constants/energyTypes.ts

Reply:
type added